kwalitee/kwalitee.py

```diff
@@ -365,9 +365,6 @@ def check_pep8(filename, **kwargs):
     :type pyflakes: bool
     :return: errors
     :rtype: `list`
-
-    .. seealso:: :py:class:`pep8.Checker`
```

Comment:
Yes, it would work. I did not want to start changing `pep8` everywhere... but I guess it may be enough to do it here...